samples/kvsWebRTCClientMasterGstreamerSample.c

```diff
@@ -78,11 +78,12 @@ GstFlowReturn on_new_sample(GstElement* sink, gpointer data, UINT64 trackid)
                 pSampleStreamingSession->videoTimestamp += SAMPLE_VIDEO_FRAME_DURATION; // assume video fps is 30
             }
 
-            status = writeFrame(pRtcRtpTransceiver, &frame);
-            if (status != STATUS_SUCCESS) {
+            if (status != STATUS_SRTP_NOT_READY_YET) {
```

Comment:
I don't understand any of this code. Who sets the status in the first place?

Reply:
The writeFrame call sets it. Earlier, we were returning `STATUS_SUCCESS` until SRTP was not set up. But, there was an issue while trying to calculate start up latency since the frames would not be sent out till SRTP was ready and application had no way of knowing it.

Reply:
Oh - it's actually right on LN 80. My bad, I was staring at it and didn't see it being set.

Suggest to combine it

if (STATUS_FAILED(status) && status != STATUS_SRTP_NOT_READY_YET) {
blah;
}

Reply:
Did not use `STATUS_FAILED` since the sample does not use any of our usual `CHK_x` or "SDK" macros. So keeping it consistent with that. But i can certainly combine the if's